### ISSUE TITLE
Indicate installed extensions that are out-of-date

### ISSUE THREAD

**Issue by flxzt** [mjakeman/extension-manager#133]
Currently there is no indication if the installed extension is out of date and they still can be activated.
I think if would be preferable to at least mark them as out-of-date, like the extension app does.
(but maybe keep the option to activate them, as not all out-of-date extension are broken after a Gnome update)

Heres a screenshot comparison:
![Screenshot from 2022-03-19 15-59-12](https://user-images.githubusercontent.com/19841886/159126364-c4c2503e-0bb5-4f84-bd63-ba914d834521.png)

**Comment by mjakeman** [comment]
This is supported in master (as of #123).

The app now properly distinguishes between the different "error", "out of date", and "enabled/disabled" states. It'll be released in 0.3, which is on track for around GNOME 42's launch date (otherwise everyone's extensions will be broken :/).

The program respects the `disable-extension-version-validation` setting (a bit more work needed in #64) which you can turn off with this command: 
```
gsettings set org.gnome.shell disable-extension-version-validation true
```

**Comment by flxzt** [comment]
Ah, great :) maybe the setting could be documented, lets say in the README or directly in the app UI? Even though its a shell setting, I could how users search for it in the extension.
